client/src/components/Doctor/DoctorAppointments.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FunctionComponent, useContext } from "react";
+import { useNavigate } from "react-router-dom";
+import { AuthContext } from "../../providers/AuthProvider";
+
+const DoctorAppointments: FunctionComponent<{}> = () => {
+  const appointments = [
+    {
+      name: "Shruthi",
+      email: "shruthi@gmail.com",
+      date: "03/26/2023",
+      time: "9.00 AM",
+      status: "Pending",
+      id: "1",
+    },
+    {
+      name: "Hasini",
+      email: "hasini@gmail.com",
+      date: "03/27/2023",
+      time: "10.00 AM",
+      status: "Confirmed",
+      id: "1",
+    },
+  ];
```

Comment:
i hope this will be integrated with backend!

Reply:
Yes. This PR is only for UI development. I will have the API integration and the functionality addressed in another ticket.